messaging/src/main/java/org/axonframework/common/Priority.java

```diff
@@ -34,10 +38,14 @@
      * Value indicating the annotated member should come last.
      */
     int LAST = Integer.MIN_VALUE;
+    /**
+     * Value indicating the annotated member should be placed at the "lower quarter".
+     */
+    int LOWER = Integer.MIN_VALUE / 2;
     /**
      * Value indicating the annotated member should be placed at the "lower half".
      */
-    int LOW = Integer.MIN_VALUE / 2;
+    int LOW = Integer.MIN_VALUE / 4;
```

Comment:
Changing this value might cause undesired side-effect when someone has used literal values to indicate a position relative to LOW. Probably unlikely, but still. Also, the javadoc seems to describe the right solution, but the implementation uses different actual values.
I would suggest LOWER to be specified as `(Integer.MIN_VALUE / 2) + (Integer.MIN_VALUE / 4)` (exactly halfway in between LOW and LAST). Similarly, a HIGHER could be used as `(Integer.MAX_VALUE / 2) + (Integer.MAX_VALUE / 4))`